src/main/java/com/github/orest/car_shop/service/DBWorkerService.java

```diff
@@ -0,0 +1,34 @@
+package com.github.orest.car_shop.service;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.sql.*;
+
+@AllArgsConstructor
+@Data
+public class DBWorkerService {
+    private static final String driverName = "com.mysql.jdbc.Driver";
+    private static final String url = "jdbc:mysql://localhost:3306/car_shop";
+    private static final String login = "root";
+    private static final String password = "root";
+
+    private Statement statement;
+
+    public DBWorkerService() {
+        try {
+            Connection connection = DriverManager.getConnection(url, login, password);
```

Comment:
You should close connection and statement. You can use try-with-resources or finally block
https://stackoverflow.com/questions/25864235/why-we-should-close-the-connection-in-jdbc-if-we-dont-do-it-what-will-happen